cfgov/cfgov/settings/base.py

```diff
@@ -89,6 +89,7 @@
     'wellbeing',
     'search',
     'paying_for_college',
+    'prepaid_agreements',
```

Comment:
`isort` will also sort for you:

```
isort --recursive cfgov
```

Reply:
I can add it there, but is there a reason those are not currently in alphabetical order?  I'm having trouble understanding what value this configuration adds.

Reply:
The .isort.cfg file is used to configure how isort works [when we run it as part of tox](https://github.com/cfpb/cfgov-refresh/blob/d4b9b329e167b78a6060ae4238f930069829500c/tox.ini#L93). Part of the import ordering config is to group imports a certain way: first built-in Python things, then Django/Wagtail, then other third-party packages, and finally our own "first-party" packages in a separate section. If `prepaid_agreements` isn't included in that file, then imports would get sorted with the other third-party packages as opposed to with our own.

You correctly point out that [the current list](https://github.com/cfpb/cfgov-refresh/blob/48c3dfe6733e25dd584bc606f4ce73d8b9f93d0b/.isort.cfg#L13-L27) in that file isn't ordered; as far as I know that order shouldn't matter and so that should be fixed for better maintainability.

Reply:
Ooh ok, thanks for the detailed explanation!

I added it in this commit - https://github.com/cfpb/cfgov-refresh/pull/5207/commits/0d02553ca3dbc3d7f349cf500c3a283a9700e020 - alphabetizing it in the first part of the list that appears alphabetized.